scripts/utils/bdlColorManager

```diff
@@ -96,8 +96,11 @@ async function main() {
     const fileType = fileName.split('.').pop();
     const verboseMode = process.argv[4] === '--verbose';
 
-    const converter = fileType === 'js' ? camelCase : identity;
-    const reBadName = new RegExp(Object.keys(conversionMap).map(entry => converter(entry)).join('|'), 'g');
+    const toRegExp = fileType === 'js' ? identity : entity => `\\\$${entity}(?!-)`;
```

Comment:
fileType === 'js' repeated 4 times :) but since this is non prod stuff its fine haha

Reply:
I'll probably go in here 1-2 more times to tighten this up, and find a way to test it too. still need to do something about the `toRegExp(caseConverter())` combo too :) ideally, anyone needing to move off old values has easy paths forward, incl. using this if needed